src/app.rs

```diff
@@ -249,12 +249,12 @@ lazy_static! {
               red, blue, green, cyan, magenta, yellow, white and black. \
               Styles are limited to nobold, bold, nointense or intense.\n\n\
               The format of the flag is {type}:{attribute}:{value}. {type} \
-              should be one of path, line or match. {attribute} can be fg, bg \
-              or style. {value} is either a color (for fg and bg) or a text \
-              style. A special format, {type}:none, will clear all color \
-              settings for {type}.\n\nFor example, the following command will \
-              change the match color to magenta and the background color for \
-              line numbers to yellow:\n\n\
+              should be one of path, line, column, match or separator. \
+              {attribute} can be fg, bg or style. {value} is either a color \
+              (for fg and bg) or a text style. A special format, {type}:none, \
+              will clear all color settings for {type}.\n\nFor example, the \
+              following command will change the match color to magenta and \
+              the background color for line numbers to yellow:\n\n\
```

Comment:
These docs also need to be added to `doc/rg.1.md` and the `doc/rg.1` file needs to be regenerated. (If it's hard for you to install pandoc, then I can do the regeneration.)